server/app.js

```diff
@@ -34,9 +33,7 @@ app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
-app.use("/", indexRouter);
-app.use("/ping", pingRouter);
-
+app.use('/user', matchRouter);
```

Comment:
```suggestion
app.use('/api/user', matchRouter);
```
It's good to add the api path here